extensions/mssql/src/utils.ts

```diff
@@ -132,6 +133,17 @@ export async function getParallelMessageProcessingConfig(): Promise<boolean> {
 	return (azdata.env.quality === azdata.env.AppQuality.dev && setting.globalValue === undefined && setting.workspaceValue === undefined) ? true : config[parallelMessageProcessingConfig];
 }
 
+export async function getEnableSqlAuthenticationProviderConfig(): Promise<boolean> {
```

Comment:
This doesn't seem to need to be async (if you could fix up the other one too that'd be great, another thing I missed cleaning up when I changed this)